ui/widgets/nuxeo-selectivity.js

```diff
@@ -2439,8 +2439,12 @@ typedArrayTags[weakMapTag] = false;
             resultsHtml = this.selectivity.template('tagExists');
           } else if (!resultsHtml && !options.add && searchText) {
             resultsHtml = this.selectivity.template('noResults', { term: options.term });
+          } else if (!resultsHtml && !options.add && !searchText) {
```

Comment:
Unneccessary code. Kindly remove it

Reply:
> `Unneccessary code. Kindly remove it`

this line of code checks whether we have a search text or not, if we don't have any search text then it will prevent unnecessary `loading...` text from all dropdowns

Reply:
than remove it(searchText) from above line. No need to add repetitive code.

Reply:
> than remove it(searchText) from above line. No need to add repetitive code.

It is required in some places, like, if a user searches a data which do not exist then we display a message with search text.

Reply:
Done.